indico/modules/events/papers/operations.py

```diff
@@ -57,3 +61,21 @@ def update_competences(user_competences, competences):
     event.log(EventLogRealm.management, EventLogKind.positive, 'Papers',
               "Updated competences for user {}".format(user_competences.user.full_name), session.user,
               data={'Competences': ', '.join(competences)})
+
+
+def paper_change_state(contribution, contrib_data, judgment, judge, send_notifications=False):
```

Comment:
Just clarifying, the rationale is to use verbs for functions/methods as much as possible.